### ISSUE TITLE
How to run the code?

### ISSUE THREAD

**Issue by 0bserver111** [youngpy996/FD-MVLLM#3]
Should I run the gen_pic.py file first, place the generated images into the specified dataset folder, and then directly run main_run.py?

**Comment by youngpy996** [comment]
> Should I run the gen_pic.py file first, place the generated images into the specified dataset folder, and then directly run main_run.py?

Yes，follow as: rwa data --> csv --> picture --> dataset folder --> main_run.py

**Comment by 0bserver111** [comment]
<img width="1055" height="314" alt="Image" src="https://github.com/user-attachments/assets/3deda954-19dc-4399-b99f-9ece68c07890" />

python main_run.py --task_name classification --model Classification --checkpoints "./checkpoints" --csv_root ".../FD-MVLLM-FD-MVLLM_v1.1/dataset/JNU-Bearing-Dataset" --image_root ".../FD-MVLLM-FD-MVLLM_v1.1/dataset/result_imgs" --llm_model_root ".../llm_models_v2/DeepSeek-R1-Distill-Llama-8B" --llm_model deepseek --llm_dim 4096 --train_epochs 50 --batch_size 8 --d_model 8 --patch_len 8 --learning_rate 0.0002 --llm_layers 2 --enc_out 1024

With approximately 45 epochs completed (starting from around the 7th epoch), the training accuracy has plateaued at only about 63%. What are the potential reasons for this low accuracy?

**Comment by youngpy996** [comment]
Based on my experience adjusting parameters at the time, the potential causes are as follows:
1. Please adjust the `llm_layers` parameter. Theoretically, a higher value yields better results, but it also imposes higher demands on computing performance.
2. Consider making appropriate adjustments to the parameters used for generating CWT output images; this significantly impacts image quality, which may in turn affect overall accuracy.
